tools/export_onnx.py

```diff
@@ -8,6 +8,8 @@
 
 import torch
 from torch import nn
+import onnx
+from onnxsim import simplify
```

Comment:
maybe this import should be placed in L90?

Reply:
In general import statements should be placed at the top, so that if a user is missing on dependency the program will show an error. In this case It's ok to move these imports to dedicated block so that user can avoid using the feature all together.